server/endpoints_v2.go

```diff
@@ -31,6 +31,10 @@ const (
 	// ClusterInfoEndpoint provides information about given cluster retrieved from AMS API
 	ClusterInfoEndpoint = "cluster/{cluster}/info"
 
+	// UpgradeRisksPredictionEndpoint returns the prediction about upgrading
+	// the given cluster.
+	UpgradeRisksPredictionEndpoint = "cluster/{cluster}/upgrade-risks-prediction"
```

Comment:
again, `clusters` with the s

Reply:
+1 :)

Reply:
Take a look into line 32. I think that we put it in singular for v2 endpoints because you can only ask for 1 cluster at a time, so the resource name is in singular